transports/transports.go

```diff
@@ -69,3 +70,13 @@ func Register(t types.ImageTransport) {
 func ImageName(ref types.ImageReference) string {
 	return ref.Transport().Name() + ":" + ref.StringWithinTransport()
 }
+
+// List returns a comma separate list of transports
+func ListNames() []string {
+	var names []string
+	for _, transport := range kt.transports {
```

Comment:
we probably need to hold the lock when iterating here as @mtrmac pointed out in the other PR